test/e2e/storage/utils/utils.go

```diff
@@ -182,6 +182,8 @@ func TestVolumeUnmountsFromDeletedPodWithForceOption(c clientset.Interface, f *f
 		err = c.CoreV1().Pods(clientPod.Namespace).Delete(clientPod.Name, &metav1.DeleteOptions{})
 	}
 	Expect(err).NotTo(HaveOccurred())
+	// Wait for pod to enter "Terminating state"
+	time.Sleep(30 * time.Second)
```

Comment:
Isn't there a helper method does something like "test condition and wait with exponential backoff", if not maybe consider for a future refactor.

Reply:
Yes. `retryWithExponentialBackOff` in _test/e2e/scalability/load.go_.
I haven't used it but it's there. Would be nice to move it to _framework_ at some point...